test/lib/scsi/scsi_bdev/scsi_bdev_ut.c

```diff
@@ -489,6 +489,17 @@ task_complete_test(void)
 	spdk_bdev_scsi_task_complete(&event);
 	CU_ASSERT_EQUAL(task.status, SPDK_SCSI_STATUS_GOOD);
 
+	bdev_io.status = SPDK_BDEV_IO_STATUS_SCSI_ERROR;
+	bdev_io.error.scsi.sc = SPDK_SCSI_STATUS_CHECK_CONDITION;
+	bdev_io.error.scsi.sk = SPDK_SCSI_SENSE_HARDWARE_ERROR;
+	bdev_io.error.scsi.asc = SPDK_SCSI_ASC_WARNING;
+	bdev_io.error.scsi.ascq = SPDK_SCSI_ASCQ_POWER_LOSS_EXPECTED;
+	spdk_bdev_scsi_task_complete(&event);
+	CU_ASSERT_EQUAL(task.status, SPDK_SCSI_STATUS_CHECK_CONDITION);
+	CU_ASSERT_EQUAL(task.sense_data[4], SPDK_SCSI_SENSE_HARDWARE_ERROR);
```

Comment:
These offsets will need to be tweaked - we moved the iSCSI SenseLength out of the generic SCSI task sense data, so these should all be adjusted by -2 (commit cc1146a8b5999f3a5718034fd8313b267d2e7aca).